### PR TITLE
add spdx fix

### PR DESCRIPTION
**Pull request by Jinzhengxu**

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -3596,8 +3596,8 @@ https://github.com/sonatype-nexus-community/nexus-repository-helm,60a9e8de0d97bf
 https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot.toolkits.graph.TestDominance.TestAcyclicCFG,ID,Accepted,https://github.com/soot-oss/soot/pull/1802,
 https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot.toolkits.graph.TestDominance.TestMultiTailedPostDom,ID,Accepted,https://github.com/soot-oss/soot/pull/1802,
 https://github.com/soot-oss/soot,23758df0ce8327bb414d982aeb325a4f5e83ccf8,.,soot.toolkits.graph.TestDominance.TestSimpleDiamond,ID,Accepted,https://github.com/soot-oss/soot/pull/1802,
-https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.tesConcludedLicensetCol,ID,,,
-https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.testDeclaredLicenseCol,ID,,,
+https://github.com/spdx/tools,cf965bc2fa99e1c5ca23f992f18cef48c08dec9f,.,org.spdx.compare.TestPackageSheet.tesConcludedLicensetCol,ID,Opened,https://github.com/spdx/tools/pull/312,
```
Comment (xieyt2000) [inline pr-data.csv:3599]: Isn't this merged?

**Review by xieyt2000 — CHANGES_REQUESTED** [review]

File: pr-data.csv (hunk 1 above)
Comment (Jinzhengxu) [reply pr-data.csv:3599]: Those two tests are flaky due to the same reason, so it refer to same PR.

Comment (darko-marinov) [reply pr-data.csv:3599]: Yuntong said your status should be `Accepted` (congratulations for that) not just `Opened`.

Comment (Jinzhengxu) [reply pr-data.csv]: Thanks for the reminder, I just saw it.

**Pull request merged**